frontend/src/pages/appTaskManagement/components/AppTaskManagement/index.js

```diff
@@ -101,6 +102,7 @@ export default function AppTaskManagement() {
 			</div>
 			<UpdateTaskModal />
 			<RemoveAllPoliciesModal />
```

Comment:
@prem056627  can you check if this RemoveAllPoliciesModal is required, if not remove it